content/03.methods.md

```diff
@@ -12,7 +12,13 @@ describe what a label function is and how many we created for each relation
 ### Generative Model
 talk about generative model and how it works
 ### Word Embeddings
-mention facebooks fasttext model and how we used it to train word vectors
+Word embeddings are representations that map individual words to k-dimensional real valued vectors.
```

Comment:
Is "k-dimensional" a broadly understood term? I see a definition at https://xlinux.nist.gov/dads/HTML/kdimensional.html, but perhaps it would be clearer to say  something like a  "user-specified dimension"